src/assessments/seo/textImagesAssessment.js

```diff
@@ -43,6 +50,8 @@ class TextImagesAssessment extends Assessment {
 		let assessmentResult = new AssessmentResult();
 		this.imageCount = researcher.getResearch( "imageCount" );
 		this.altProperties = researcher.getResearch( "altTagCount" );
```

Comment:
The test coverage for this researcher (imageAltTags) is 92%. Please check if you can add specs.